roles/openshift_master/tasks/main.yml

```diff
@@ -36,14 +36,9 @@
   command: systemctl daemon-reload
   when: install_result | changed
 
-- name: Create certificate parent directory if it doesn't exist
-  file:
-    path: "{{ openshift_cert_parent_dir }}"
-    state: directory
-
 - name: Create config parent directory if it doesn't exist
   file:
-    path: "{{ openshift_master_config | dirname }}"
+    path: "{{ openshift_master_config_dir | dirname }}"
```

Comment:
This should actually be "{{ openshift_master_config_dir }}"  using the dirname filter is only ensuring that /etc/openshift exists and not /etc/openshift/master